hiyapyco/__init__.py

```diff
@@ -327,11 +345,23 @@ def _simplemerge(self, a, b):
         if self.dereferenceyamlanchors:
             a = copy.deepcopy(a)
             b = copy.deepcopy(b)
-        logger.debug('simplemerge %s (%s) and %s (%s)' % (a, type(a), b, type(b),))
-        # FIXME: make None usage configurable
+        logger.debug(
+            "simplemerge %s (%s) and %s (%s)"
+            % (
+                a,
+                type(a),
+                b,
+                type(b),
+            )
+        )
         if b is None:
```

Comment:
would you mind changing it to something like
```diff
-           logger.debug('pass as b is None')
            # override -> None replaces object, no matter what.
            if self.none_behavior == NONE_BEHAVIOR_OVERRIDE:
+               logger.debug('b is None + none_behavior in use => return None')
                return None
            # default behavior is to attempt merge or fail
-            else:
+            logger.debug('pass as b is None')
-               pass
+            pass
```
avoids a unnecessary `else` and improves debug output ...

Reply:
Makes sense! 

btw, when using 

````
```suggestion
```
````

![image](https://github.com/user-attachments/assets/cfbb4f91-5b1f-4b06-84ef-fc84feb47946)


instead of 
````
```diff
```
````

one can directly apply the changes with one click. 
Because you were wondering, [here](https://github.com/zerwes/hiyapyco/pull/76#issuecomment-2317209443) ;)

Reply:
Thank you for the hint ... looks like not only my python coding style, but my github knowledge too is more then slightly outdated :-1: (probably much more stuff including my self, but this is another story ...)